SydTek.md

```diff
@@ -0,0 +1,228 @@
+
+- https://github.com/kingassune
+- https://github.com/HashWarlock
+- https://github.com/jgophd
+
+### Team LinkedIn Profiles
+
+- https://www.linkedin.com/in/justin-goldston/
+- https://www.linkedin.com/in/justyna-osowska/
+- https://www.linkedin.com/in/jordichaffer/
+- https://www.linkedin.com/in/dontrail-cotlage-9312a660/
+- https://www.linkedin.com/in/joshua-waller-33935773/
+- https://www.linkedin.com/in/suyeon-han/
+
+## Development Status :open_book:
+
+- academic publications relevant to the problem we wrote that will be published in October of 2020: https://bit.ly/3RnCmMy
+
+- References to conversations you might have had related to this project with anyone from the Web3 Foundation,
+A message from David Hawig on 6/7/22.
+
+Hi Justin,
+
+Sorry for the late reply. Regarding souldbound tokens and social recovery, that’s actually something we already have in our ecosystem with rmrk (see  
+https://twitter.com/bitfalls/status/152929818658155725) and the social recovery pallet on kusama (https://github.com/paritytech/substrate/tree/master/frame/recovery). But it would be interesting for someone to actually look into it and explore it in more detail and potential future applications. Also most people are probably not aware that this already exists on kusama.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 3 months
+- **Full-Time Equivalent (FTE):**  3FTE, the listed members would contribute to different deliverables based on their skill-set.
+- **Total Costs:** 29,964 USDC 
+
+### Milestone 1 — State of the Industry on soulbound tokens and digital inheritance 
+
+- **Estimated duration:** 1 month
+- **FTE:**  3
+- **Costs:** 10,000 USDC - link to budget details outlined in the Additional Information Section
+
+| Number | Deliverable       | Specification                                                                                                                                                                                                                                                                                                                  |
+| ------: | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+|      0. | Rights            | All developed materials will be publicly accessible and public domain. Milestone 1 is research oriented and as such there is no code to test.                                                                                                                                                                                                                                                         |
```

Comment:
How are you going to make these accessible? By "developed materials" (here and below), I assume you also mean the raw data?